src/CustomChord.ts

```diff
@@ -1,15 +1,18 @@
+import CustomOscillatorType from "./CustomOscillatorType";
 import {Note} from "./Note";
 import PlayableInterface from "./PlayableInterface";
 
 class CustomChord implements PlayableInterface {
     /**
      * @param {Note[]} notes - The notes that will be played simultaneously.
+     * @param {number} duration - How long the chord will be played in seconds. (you most likely want to stay under 1 second)
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏Remove duration here and in the constructor